src/deadline_worker_agent/startup/entrypoint.py

```diff
@@ -82,7 +83,13 @@ def entrypoint(cli_args: Optional[list[str]] = None, *, stop: Optional[Event] =
         config.log()
 
         # Register the Worker
-        worker_bootstrap = bootstrap_worker(config=config)
+        try:
+            worker_bootstrap = bootstrap_worker(config=config)
+        except NoRegionError:
+            _logger.warn(
+                "The Worker Agent was started with no AWS region specified. Refer to the Deadline Cloud Worker Agent documentation for guidance."
+            )
```

Comment:
Should we include the URL to the README? I think we'd want

https://github.com/aws-deadline/deadline-cloud-worker-agent/blob/release/README.md#running-outside-of-an-operating-system-service

Reply:
Added, thanks